src/synthesizer/particle/stars.py

```diff
@@ -1788,8 +1788,14 @@ def get_particle_line_attenuated(
                 tau_v_stellar, intrinsic_line._wavelength
             )
 
+            if not np.isscalar(T_nebular):
```

Comment:
I don't think the logic is needed here. The transmission is guaranteed to be returned in the strange shape that was causing issues. 

Porbably better to have `get_transmission` to take some sort of shape argument to handle this situation rather than fix it outside that method. That way we won't have to play this game anywhere else we have this problem.

Reply:
It's probably a simple case of modifying `get_transmission` to remove the wavelength axis when a single wavelength is passed.

Reply:
OK, I agree. I've fixed `get_transmission` instead to return the correct shape depending on whether tau_v and lam are arrays.